### ISSUE TITLE
NVDA not announcing border styles in MS Word tables

### ISSUE THREAD

**Issue by andrew-l-d** [nvaccess/nvda#13865]
You have arrived at a default issue template (bug report).
It is preferable to [choose the type of issue](https://github.com/nvaccess/nvda/issues/new/choose) to submit.
Hint: Swap to the preview tab to be able to click the links.
Direct links to the templates:
- [Bug report](
- https://github.com/nvaccess/nvda/issues/new?template=bug_report.md)
- [Feature request](https://github.com/nvaccess/nvda/issues/new?template=feature_request.md)

### Steps to reproduce:
In NVDA's Document Formatting setting set table borders to both style and colour
In MS Word create a table
Navigate the table
Use NVDA's announce formatting feature (both caret and review cursor)

### Actual behavior:
No information about cell border style or colour is presented
### Expected behavior:
Border style and colour would be presented by NVDA.  Especially when reviewing the most helpful Microsoft Word with NVDA document, I suspect this is not a bug and that NVDA cannot provide this information in Word or HTML files, but would appreciate feedback.
#### NVDA installed/portable/running from source:
NVDA installed
#### NVDA version:
2022.1
#### Windows version:
Windows 10
#### Name and version of other software in use when reproducing the issue:
MS Word (Office 365)
#### Other information about your system:
Cannot think of anything relevant.  Tried on two computers with the same result.  
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
NVDA 2020.4 and NVDA 2022.2 beta 3
Behaviour consistent with 2022.1
#### If add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes
